### PR TITLE
misc: update deps

### PR DESCRIPTION
**Pull request by MarcelCoding**

### CODE REVIEW

File: nix/frontend.nix [hunk 1]
```diff
@@ -1,11 +1,7 @@
-{ callPackage, lib, path, stdenv, nodejs, baseHref ? "/", title ? "NuschtOS Search" }:
+{ lib, stdenv, nodejs, pnpm, baseHref ? "/", title ? "NuschtOS Search" }:
 
 let
   manifest = lib.importJSON ../package.json;
-  pnpm = callPackage (path + "/pkgs/development/tools/pnpm/generic.nix") {
```
Comment (MarcelCoding) [inline nix/frontend.nix:5]: nixos-24.05 is now deprecated

Comment (MarcelCoding) [reply nix/frontend.nix:5]: Now or wait a few weeks?

Comment (SuperSandro2000) [reply nix/frontend.nix:5]: Let's remove it now

**Comment by SuperSandro2000** [comment]
The commit title is wrong 😅

**Comment by SuperSandro2000** [comment]
The last one where you removed 24.05, not 24.11

**Pull request merged**